lib/stringtheory/src/lib.rs

```diff
@@ -111,17 +97,17 @@ impl MetaString {
     /// This does not allocate.
     pub const fn empty() -> Self {
         Self {
-            inner: Inner::Shared(bytes::Bytes::new()),
+            inner: Inner::Owned(String::new()),
         }
     }
 
     /// Creates a new `MetaString` from the given static string.
     ///
     /// This does not allocate.
     pub fn from_static(s: &'static str) -> Self {
-        Self::try_inline(s).unwrap_or_else(|| Self {
-            inner: Inner::Shared(bytes::Bytes::from_static(s.as_bytes())),
-        })
+        Self {
+            inner: Inner::Static(s),
```

Comment:
I changed this because there's no good reason to inline it if we don't have to, as it should be equivalent to following the static reference directly.